deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -304,18 +405,19 @@ spec:
           type: ""
       - name: config-volume
         configMap:
-          name: fluentd-config
+          name: fluentd
       containers:
       - name: fluentd
-        image: sumologic/kubernetes-fluentd:0.1.0
+        image: "sumologic/kubernetes-fluentd:0.0.0"
```

Comment:
should be `0.1.0` .. Also could we avoid the quotes somehow? I don't normally see yaml files with these image values quoted.